src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -281,6 +297,15 @@ public static Runspace CreateRunspace(PSHost psHost, PSLanguageMode languageMode
             // should have the same LanguageMode of whatever is set by the system.
             initialSessionState.LanguageMode = languageMode;
 
+            // We set the process scope's execution policy (which is really the runspace's scope) to
+            // Bypass so we can import our bundled modules. This is equivalent in scope to the CLI
+            // argument `-Bypass`, which (for instance) the extension passes. Thus we emulate this
+            // behavior for consistency such that unit tests can pass in a similar environment.
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                initialSessionState.ExecutionPolicy = ExecutionPolicy.Bypass;
```

Comment:
@dkattan So we resurrected the `SetExecutionPolicy()` function because on closer inspection, the actual issue was that in the _unit test_ scenario we weren't having `Bypass` set (because the codebase's expectation is that the startup script is run with `-ExecutionPolicy Bypass`, like what the extension does), and the `SetExecutionPolicy()` actually sets the execution policy _back_ to the original inherited scoped policy, not the process policy. That is, it override's the process/runspace's policy after everything is loaded to be equivalent to the user's machine/user policy.

Reply:
@rjmholt I might rename that function `ResetExecutionPolicy()` 😆

Reply:
That works. I'm a fan of removing large chunks of code, as long as it doesn't change expected functionality :D